airflow/providers/mongo/hooks/mongo.py

```diff
@@ -90,15 +85,25 @@ def get_conn(self) -> MongoClient:
             options.update({"ssl_cert_reqs": CERT_NONE})
 
         self.client = MongoClient(self.uri, **options)
-
         return self.client
 
-    def close_conn(self) -> None:
-        """Closes connection"""
-        client = self.client
-        if client is not None:
-            client.close()
-            self.client = None
+    def _create_uri(self) -> str:
+        """
+        Create URI string from the given credentials.
+        :return: URI string.
+        """
+        srv = self.extras.pop("srv", False)
+        scheme = "mongodb+srv" if srv else "mongodb"
+        login = self.connection.login
+        password = self.connection.password
+        if login is not None and password is not None:
+            login = login.encode()
+            password = password.encode()
```

Comment:
I don’t think these are needed? `quote_plus` works with strings just fine, from my understanding.

Reply:
Unfortunately, it doesn't work fine with the normal strings, that's why I've implemented string encoding.

Reply:
How does it not work fine? An example would be nice.

Reply:
Sorry for confusing, It works fine with `quote_plus`. I forgot, that I had the encoding errors only with `quote` function.